docs/middleware/builtin/jsx-renderer.md

```diff
@@ -159,6 +159,10 @@ app.route('/blog', blog)
 `useRequestContext()` returns an instance of Context.
 
 ```tsx
+import { useRequestContext, jsxRenderer } from 'hono/jsx-renderer'
+
+const app = new Hono().use(jsxRenderer())
```

Comment:
This is my favorite, but I like to write it as follows. Other parts are written this way too.

```ts
const app = new Hono()
app.use(jsxRenderer())
```